services/src/test/java/org/apache/druid/server/AsyncQueryForwardingServletTest.java

```diff
@@ -127,7 +127,11 @@ public class AsyncQueryForwardingServletTest extends BaseJettyTest
 {
   private static int port1;
   private static int port2;
-
+  private String errorMessage;
+  private ObjectMapper mockMapper;
+  private HttpServletRequest request;
+  private HttpServletResponse response;
+  private ServletOutputStream outputStream;
```

Comment:
Please add a line after this.

```suggestion
  private ServletOutputStream outputStream;
  
```